tests/unit/authenticators/torii-test.js

```diff
@@ -44,8 +44,8 @@ describe('ToriiAuthenticator', () => {
         });
 
         it('returns a promise that resolves with the session data', function() {
-          return authenticator.restore({ some: 'data', provider: 'provider' }).then((data) => {
-            expect(data).to.eql({ some: 'other data', provider: 'provider' });
+          return authenticator.restore({ some: 'data', provider: 'provider', other: 'unrelated data' }).then((data) => {
+            expect(data).to.eql({ some: 'other data', provider: 'provider', other: 'unrelated data' });
```

Comment:
In this case, `other` is part of the data resolved by a custom `authenticator.authenticate()` method.

I'll put an example together shortly.